src/Verify.MSTest.SourceGenerator/Parser.cs

```diff
@@ -36,4 +37,16 @@ SyntaxKind.StructDeclaration or
 
         return parents.ToArray();
     }
+
+    static bool HasExistingTestContextProperty(INamedTypeSymbol typeSymbol)
```

Comment:
This appears to be dead code now. Can you remove this helper and the corresponding property in `ClassToGenerate`?

If not, what's the use case for it now?

Thanks!

Reply:
Oops yeah. I was experimenting with a possibly different diagnostic, but CS0108 will do.